e2e/framework/manager/rook_test_infra.go

```diff
@@ -271,7 +272,9 @@ func createK8sRookOperator(k8sHelper *utils.K8sHelper, tag string) error {
 	if err != nil {
 		return err
 	}
-
+	if !bytes.Contains(raw, []byte(rookOperatorImagePodSpecTag)) {
```

Comment:
we are using bytes[] since the ioutils.ReadFile returns a byte[]. We would like to keep it as a byte[] so we know where are checking or replacing contents of a file.